platforms/core-runtime/base-asm/build.gradle.kts

```diff
@@ -15,13 +15,12 @@
  */
 
 plugins {
-    id("gradlebuild.distribution.api-java")
+    id("gradlebuild.distribution.implementation-java")
```

Comment:
💭 This is not API project... So i changed it to implementationa

Reply:
Nice finds. 🤔 I wonder how we could shake these out automatically...